testing/MLDB-724-time-arithmetic.py

```diff
@@ -794,6 +794,24 @@ def test_MLDB_1509(self):
             ]
         )
 
+    @unittest.expectedFailure
+    def test_mldbfb_441_distinct_timestamp_keep_proper_timestamp(self):
+        ds = mldb.create_dataset({'id' : 'ds', 'type' : 'sparse.mutable'})
+        ds.record_row('row1', [['colA', 1, 0], ['colA', 1, 5]])
+        ds.commit()
+        res = mldb.get('/v1/query',
+                       q="SELECT distinct_timestamps(colA) FROM ds")
+        mldb.log(res)
+        self.assertFullResultEquals(res.json(), [{
+            "rowName": "row1",
+            "columns": [
+                ["distinct_timestamps(colA).0",
+                 {"ts":"1970-01-01T00:00:00Z"},
+                 "1970-01-01T00:00:00Z"],  # <--- MLDB returns the 5th second
```

Comment:
This is expected.  We returned the list of known distinct timestamp at the latest timestamp.